Task 3/Prefill An Array.js

```diff
@@ -0,0 +1,9 @@
+function prefill(length, value) {
+    if (isNaN(parseInt(length)) || length < 0 || length % 1 !== 0) {
```

Comment:
Isn't the variable `length` already a number ? The usage of `parseInt` isn't required as you are already applying logical operations on  `length` variable

Reply:
Also format the files before committing the changes

Reply:
Isn't the variable length already a number ? The usage of parseInt isn't required as you are already applying logical operations on length variable.
Actually length can be a number or anything, I have to throw an error if its not a number